view/frontend/web/js/view/payment/method-renderer/razorpay-method.js

```diff
@@ -153,6 +161,85 @@ define(
                 });
             },
 
+            _prepareHiddenFormFields: function (options, form) {
+                var self = this;
+
+                function visitNestedOption(options, parentKey) {
```

Comment:
Can we rename this function with "ceateInputFieldsFromOptions"

Reply:
Sure.